src/pages/home.jsx

```diff
@@ -111,13 +106,30 @@ function Home() {
 
       </div>
       <div className="card">
-        placegholder for carousel with hero content
+        {/* placegholder for carousel with hero content */}
+        <MskHero
+          theme="segment" 
+          img={{
+            large: 'https://www.mskcc.org/sites/default/files/2021-01/hp_hero_shah_1920x640-1.jpg',
+            medium: 'https://www.mskcc.org/sites/default/files/2021-01/hp_hero_shah_1200x400.jpg',
+            small: 'https://www.mskcc.org/sites/default/files/2021-01/hp_hero_shah_1200x400.jpg',
+            xsmall: 'https://www.mskcc.org/sites/default/files/2021-01/hp_hero_shah_1200x400.jpg',
+          }}
+          isNarrow={false}
+          title="COVID-19 Vaccination Information for Patients"
+          // title2="title 2"
+          children={heroChildren}
+          ctaLinks={[{ text: "Read More", href: "/coronavirus/covid-19-vaccine" }]}
+
+        >
+
+        </MskHero>
```

Comment:
We can simplify this call and make it more clear by putting the children inside the tag instead of being called as a prop.

```suggestion
          ctaLinks={[{ text: "Read More", href: "/coronavirus/covid-19-vaccine" }]}
        >
          {heroChildren}
        </MskHero>
```